docs/imviz/import_data.rst

```diff
@@ -0,0 +1,34 @@
+.. _imviz-import-data:
+
+***********
+Import Data
+***********
+
+Imviz can load data in the form of a filename (string), .jpg, .png,
+an astropy object, or a numpy array if the data is 2D.
+
+Importing data through the GUI
+------------------------------
+
+The first way that users can load their data into the Imviz application is
+by clicking the :guilabel:`Import` button at the top left of the application's
+user interface. This opens a dialogue where the user can enter the path of file
+that can be parsed as a :class:`~astropy.nddata.NDData` (2D data only), :class:`~astropy.io.fits.HDUList`,
+or :class:`~astropy.io.fits.ImageHDU` in the text field:
+
+.. image:: ../cubeviz/img/cubeviz_import_data.png
```

Comment:
I think @orifox said no need for screenshot. It is misleading to show Cubeviz here. Imviz user will see a different background and then they will be confused. If you must, refer back to relevant Cubeviz section with `:ref:` (as you have done below) but remove the image embedding.

```suggestion
or :class:`~astropy.io.fits.ImageHDU` in the text field.
```